src/transaction/log_manager.c

```diff
@@ -12471,6 +12471,144 @@ cdc_check_if_schema_changed (RECDES * recdes, HEAP_CACHE_ATTRINFO * attr_info)
   return or_rep_id (recdes) != attr_info->last_classrepr->id;
 }
 
+static int
+cdc_get_attribute_size (DB_VALUE * value)
+{
+  assert (!DB_IS_NULL (value));
+
+  int size = 0;
```

Comment:
Is the variable 'size' related with attribute size?
If then, What about replace `attr_size` instead of `size`?

(Not critical, and while reading through the code, I can guess 'size' is `attr_size`)

Reply:
Ok. I understand it